### ISSUE TITLE
Conflict with autoslide

### ISSUE THREAD

**Issue by ratnanil** [McShelby/reveal-helpbutton#2]
very handy plugin!  Regrettably, the help button does not play nicely with [the autoslide button](https://revealjs.com/auto-slide/), if this option is activated (see screenshot)

![DeepinScreenshot_select-area_20220322072954](https://user-images.githubusercontent.com/12532091/159421389-804d8477-db89-491a-9de4-30f7059705b3.png)

**Comment by McShelby** [comment]
You're right, this doesn't look nice. But at the moment I do not plan to fix this.

The best thing for now is to deactivate the help button when autoslides config is enabled.

Nevertheless, feel free to submit a PR if you have a solution for this
